synthetics/test_schema.go

```diff
@@ -16,17 +16,18 @@ func makeTestSchema(mode schemaMode) map[string]*schema.Schema {
 		},
 		"name": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Required: requiredOnCreate(mode),
 			Computed: computedOnRead(mode),
 		},
 		"type": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
```

Comment:
Test `type` is actually required on create.

Reply:
Addressed: a2e73bcd0db6697b5392fdb69a572f74228bc594

Reply:
Done